doc/user_guide/Parameter_Types.ipynb

```diff
@@ -587,7 +587,7 @@
     "- `param.List`: A Python list of objects, usually of a specified type.\n",
     "- `param.HookList`: A list of callable objects, for executing user-defined code at some processing stage\n",
     "\n",
-    "List Parameters accept a Python list of objects. Typically the `item_type` will be specified for those objects, so that the rest of the code does not have to further check types when it refers to those values. Where appropriate, the `bounds` of the list can be set as (_min_length_, _max_length_), defaulting to `(0,None)`. Because List parameters already have an empty value ([]), they do not support `allow_None`.\n",
+    "List Parameters accept a Python list of objects. Typically the `item_type` will be specified for those objects, so that the rest of the code does not have to further check types when it refers to those values. Where appropriate, the `bounds` of the list can be set as (_min_length_, _max_length_), defaulting to `(0,None)`. Because List parameters already have an empty value ([]), they do not support `allow_None`. `is_isntance` defaults to `True` and can be set to `False` to declare the objects must be subclasses of `item_type`.\n",
```

Comment:
Typo: `is_isntance`

Reply:
Thanks, fixed in https://github.com/holoviz/param/pull/1023/commits/8a4dd74f6da15277688a8099a25ba6446a291714